components/icons/VerifiedCheck.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <svg
+    width="17"
+    height="16"
+    viewBox="0 0 17 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M4.00629 14.9282C4.68371 15.3193 5.47995 15.3181 6.13199 14.9951C6.53517 15.6009 7.22415 16 8.00635 16C8.78854 16 9.47752 15.6009 9.8807 14.9951C10.5327 15.3181 11.329 15.3193 12.0064 14.9282C12.6838 14.5371 13.0809 13.847 13.1272 13.1208C13.8533 13.0745 14.5435 12.6774 14.9346 12C15.3257 11.3226 15.3245 10.5264 15.0015 9.87434C15.6072 9.47116 16.0063 8.78219 16.0063 8C16.0063 7.21781 15.6072 6.52885 15.0015 6.12566C15.3245 5.47364 15.3257 4.67741 14.9346 4.00001C14.5435 3.32259 13.8533 2.92549 13.1271 2.8792C13.0808 2.15304 12.6837 1.46289 12.0063 1.0718C11.3289 0.680709 10.5327 0.681878 9.88068 1.00483C9.47749 0.399126 8.78853 0 8.00635 0C7.22416 0 6.53519 0.399131 6.13201 1.00484C5.47998 0.681875 4.68376 0.6807 4.00637 1.07179C3.32896 1.46289 2.93187 2.15304 2.88556 2.8792C2.1594 2.92552 1.46925 3.32261 1.07816 4.00001C0.687065 4.6774 0.688238 5.47363 1.0112 6.12565C0.405484 6.52884 0.00634766 7.21781 0.00634766 8C0.00634766 8.7822 0.405488 9.47117 1.0112 9.87436C0.688235 10.5264 0.687056 11.3226 1.07815 12C1.46924 12.6774 2.15936 13.0745 2.88549 13.1208C2.9318 13.847 3.3289 14.5371 4.00629 14.9282ZM11.7071 6.70711C12.0977 6.31658 12.0977 5.68342 11.7071 5.29289C11.3166 4.90237 10.6834 4.90237 10.2929 5.29289L7.00003 8.58579L5.70714 7.29289C5.31662 6.90237 4.68345 6.90237 4.29293 7.29289C3.9024 7.68342 3.9024 8.31658 4.29293 8.70711L6.29293 10.7071C6.68345 11.0976 7.31662 11.0976 7.70714 10.7071L11.7071 6.70711Z"
+      fill="#FF5C00"
```

Comment:
@AndyClifford this one can be `current-color` and then you can use `text-brand-accent` 👍